inbm/Changelog.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 ### Fixed
+ - HSD 521500      - Fixed ModuleNotFoundError: No module named 'inbm_common_lib'
  - RTC 520951      - Remove references to 'inbm-lib' from requirements.txt in intel-inb-manageability repository
```

Comment:
Instead of putting what was fixed, but in what was changed.  State that the Pyinstaller version was updated to 5.6.2

Reply:
This should say 'fixed build error by updating PyInstaller to 5.6.2'